Sources/BrowserServicesKit/UserScript/ContentScopeUserScript.swift

```diff
@@ -24,13 +24,22 @@ public final class ContentScopeProperties: Encodable {
     public let globalPrivacyControlValue: Bool
     public let debug: Bool = false
     public let sessionKey: String
+    public let platform = ContentScopePlatform()
 
     public init(gpcEnabled: Bool, sessionKey: String) {
         self.globalPrivacyControlValue = gpcEnabled
         self.sessionKey = sessionKey
     }
 }
 
+public struct ContentScopePlatform: Encodable {
+    #if os(macOS)
+    let name = "macos"
+    #else
+    let name = "ios"
```

Comment:
There are watchOS and tvOS so if you would like to have this 100 reliable, you can add `#elseif os(iOS) ` and then name = "unknown" for the `#else`. But that's not necessary since we don't use BSK for other platforms

Reply:
@tomasstrba whilst not strictly necessary as you say, I do prefer the explicitness of your suggest, I will update :)

Reply:
@tomasstrba please see the changes :)

Reply:
👍 awesome